src/main/java/hudson/plugins/sauce_ondemand/SauceOnDemandBuildWrapper.java

```diff
@@ -202,7 +205,8 @@ public SauceOnDemandBuildWrapper(
             boolean launchSauceConnectOnSlave,
             boolean useOldSauceConnect,
             boolean verboseLogging,
-            boolean useLatestVersion
+            boolean useLatestVersion,
+            boolean useGeneratedTunnelIdentifier
```

Comment:
then let's delete the extra setting and always use the generated TunnelID?
then code becomes even simpler